crates/test-cluster/src/indexer_util.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use jsonrpsee::http_client::{HttpClient, HttpClientBuilder};
+use std::path::PathBuf;
+use std::time::Duration;
+use sui_config::local_ip_utils::get_available_port;
+use sui_indexer::tempdb::TempDb;
+use sui_indexer::test_utils::{
+    start_indexer_jsonrpc_for_testing, start_indexer_writer_for_testing,
+};
+use sui_sdk::{SuiClient, SuiClientBuilder};
+use tempfile::TempDir;
+use tokio::time::sleep;
+
+pub(crate) struct IndexerHandle {
+    #[allow(unused)]
+    cancellation_tokens: Vec<tokio_util::sync::DropGuard>,
+    #[allow(unused)]
+    data_ingestion_dir: Option<TempDir>,
+    #[allow(unused)]
+    database: TempDb,
+}
+
+// TODO: this only starts indexer writer, reader and jsonrpc server today.
```

Comment:
nit: this reads like reader and json rpc server as separate instances, 
```suggestion
// TODO: this only starts indexer writer and reader (jsonrpc server) today.
```